### PR TITLE
[Installer] - Add SeLib.dll in installer

### PR DESCRIPTION
**Pull request by ivandrofly**
# Status
- [x] Test Installer
- [x] Test Uninstaller
- [ ] Ready to merge

### CODE REVIEW

File: installer/Subtitle_Edit_installer.iss [hunk 1]
```diff
@@ -170,107 +170,106 @@ Name: quicklaunchicon;    Description: {cm:CreateQuickLaunchIcon}; GroupDescript
...
+Source: "..\Dictionaries\ru_RU_user.xml"; DestDir: "{userappdata}\Subtitle Edit\Dictionaries"; Flags: ignoreversion onlyifdoesntexist uninsneveruninstall; Components: main
+Source: "..\Dictionaries\rus_OCRFixReplaceList.xml"; DestDir: "{userappdata}\Subtitle Edit\Dictionaries"; Flags: ignoreversion onlyifdoesntexist uninsneveruninstall; Components: main
+Source: "..\Dictionaries\spa_OCRFixReplaceList.xml"; DestDir: "{userappdata}\Subtitle Edit\Dictionaries"; Flags: ignoreversion onlyifdoesntexist uninsneveruninstall; Components: main
+Source: "..\Dictionaries\srp_OCRFixReplaceList.xml"; DestDir: "{userappdata}\Subtitle Edit\Dictionaries"; Flags: ignoreversion onlyifdoesntexist uninsneveruninstall; Components: main
+Source: "..\Dictionaries\swe_OCRFixReplaceList.xml"; DestDir: "{userappdata}\Subtitle Edit\Dictionaries"; Flags: ignoreversion onlyifdoesntexist uninsneveruninstall; Components: main
+Source: "{#bindir}\Hunspellx64.dll"; DestDir: "{app}"; Flags: ignoreversion; Components: main
+Source: "{#bindir}\Hunspellx86.dll"; DestDir: "{app}"; Flags: ignoreversion; Components: main
+Source: "{#bindir}\libse.dll"; DestDir: "{app}"; Flags: ignoreversion; Components: main
```
Comment (ivandrofly) [inline installer/Subtitle_Edit_installer.iss]: This should include libse.dll to installer

**Comment by ivandrofly** [comment]
@niksedk, @xylographe **SeLib.dll** should now be included in installer if you have *Inno* installed I recommend running test too..
run

**Comment by niksedk** [comment]
selib.dll should not be included in the installer - it's included inside SubtitleEdit.exe via the ilrepack in build.bat. 
The installer works now - sorry. It just failed yesterday... don't know why.

**Comment by ivandrofly** [comment]
ah okay, check https://github.com/SubtitleEdit/subtitleedit/issues/1201

**Pull request closed without merging**